static/app/views/performance/traceDetails/transactionBar.tsx

```diff
@@ -397,9 +397,9 @@ class TransactionBar extends Component<Props, State> {
           showDetail={showDetail}
           onClick={this.toggleDisplayDetail}
         >
-          <GuideAnchor target="trace_view_guide_row" disabled={!hasGuideAnchor}>
-            {this.renderTitle(scrollbarManagerChildrenProps)}
-          </GuideAnchor>
+          {/* <GuideAnchor target="trace_view_guide_row" disabled={!hasGuideAnchor}> */}
+          {this.renderTitle(scrollbarManagerChildrenProps)}
+          {/* </GuideAnchor> */}
```

Comment:
the test for this is failing in CI, but not when i run the tests in isolation locally, so i'm just trying to poke CI and see what's different. This didn't work :(